application/frontend/web/index.php

```diff
@@ -1,16 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
 
-require(__DIR__ . '/../../vendor/autoload.php');
-require(__DIR__ . '/../../vendor/yiisoft/yii2/Yii.php');
-require(__DIR__ . '/../../common/config/bootstrap.php');
-require(__DIR__ . '/../config/bootstrap.php');
+$application = require('load-app.php');
```

Comment:
Unfortunately, I think we may need to keep all this code here in `index.php` rather than another file here in the `web` folder, because people could directly access the new `web/load-app.php` file instead. What was the thought for pulling this out? Was it for getting Yii stuff available to the Behat tests?